rust/worker/src/execution/operators/knn.rs

```diff
@@ -26,6 +26,21 @@ impl PartialOrd for RecordDistance {
     }
 }
 
+/// The `KnnOperator` searches for the nearest neighbours of the specified embedding
+/// It can take different types of input and produce different types of output
```

Comment:
"It can take different types of input and produce different types of output" I don't really understand the information content of this string.

Reply:
I updated the comments to mention the two implementation of the `Operator<I, O>` trait for `KnnOperator`